rest-api/src/main/java/gov/cms/qpp/conversion/api/services/DbServiceImpl.java

```diff
@@ -50,6 +53,25 @@
 		return actOnItem(meta);
 	}
 
+	/**
+	 * Queries the DynamoDB table for unprocessed {@link Metadata}
+	 *
+	 * @return {@link List} of unprocessed {@link Metadata}
+	 */
+	public List<Metadata> getUnprocessedCpcPlusMetaData() {
+		HashMap<String, AttributeValue> valueMap = new HashMap<>();
+		valueMap.put(":cpcValue", new AttributeValue().withN("1"));
+		valueMap.put(":cpcProcessedValue", new AttributeValue().withN("0"));
+
+		DynamoDBScanExpression metadataScan = new DynamoDBScanExpression()
+				.withFilterExpression("Cpc = :cpcValue and CpcProcessed = :cpcProcessedValue")
+				.withExpressionAttributeValues(valueMap);
+
+		List<Metadata> unprocessedCpcPlusMetadata = mapper.scan(Metadata.class, metadataScan);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "unprocessedCpcPlusMetadata". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)